src/utils/UriUtils.ts

```diff
@@ -0,0 +1,42 @@
+export interface Ror2Uri {
+    scheme: "ror2mm";
+    version: string;
+    action: string;
+    parameters: string[];
+}
+
+const ror2UriRe = new RegExp(/ror2mm:\/\/(\w+)\/(\w+)\/?(.*\/?)/);
```

Comment:
Non-blocker, but perhaps a comment could be associated with a few examples of how the scheme is currently used to make reading this easier?